web/templates/web3.html

```diff
@@ -0,0 +1,50 @@
+{{ template "header.html" . }}
```

Comment:
@Hmac512 @opula Here's the file that I'm worried about. This is being served by Go, there's no bundler of any sort in sight.

Reply:
If you need a js bundler there is: [golang esbuild](https://github.com/evanw/esbuild). But I feel that this is meant to be pretty terse and simple since it is a single login page, ie. not a whole web app with lots of resources.

Reply:
I talked to those guys briefly—I think we are going to leave this be for now and revisit later. I should make a ticket.

Big fan of esbuild.